tokens.xml

```diff
@@ -1629,6 +1654,22 @@ Whenever this creature attacks, target attacking creature gains flying until end
             <token>1</token>
             <tablerow>2</tablerow>
         </card>
+        <card>
+            <name>Cat Warrior Token</name>
```

Comment:
this is a duplicate of an existing token, it is sorted alphabetically, ie W after T